phpstan-baseline.neon

```diff
@@ -140,6 +140,11 @@ parameters:
 			count: 1
 			path: src/Uri.php
 
+		-
+			message: "#^Method GuzzleHttp\\\\Psr7\\\\Uri\\:\\:escapeQueryKey\\(\\) should return string but returns string\\|null\\.$#"
```

Comment:
Why do you silence this?

Reply:
Because it's a false positive about the string replacement?

Reply:
For same reason it's silenced for `filterPath` above. In theory `preg_replace_callback` returns `null` error. I can't imagine any error that can occur here. Regexp is valid, and input is guaranteed to be string by type hinting.